modules/auxiliary/scanner/emby_ssrf_scanner.rb

```diff
@@ -0,0 +1,105 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Scanner
+  include Msf::Auxiliary::Report
+
+  def initialize
+    super(
+      'Name' => 'Emby SSRF HTTP Scanner',
+      'Description' => 'Generates a `GET` request to the provided web servers and executes an SSRF against the targeted EMBY server. Returns the server header, HTML title attribute and location header (if set). This is useful for rapidly identifying web applications on the internal network using the Emby SSRF vulnerability (CVE-2020-26948).',
+      'Author' => 'Btnz',
+      'License' => MSF_LICENSE,
+      'Disclosure Date' => 'September 1 2020',
+      'References' =>
+              [
+                ['CVE', '2020-26948'],
+                ['URL', 'https://github.com/btnz-k/emby_ssrf']
+              ]              
+    )
+
+    deregister_options('VHOST', 'RPORT', 'SNAPLEN', 'SSL')
+
+    register_options(
+      [
+        OptBool.new('STORE_NOTES', [true, 'Store the information in notes. Use "notes -t http.title" to view', true]),
+        OptBool.new('SHOW_TITLES', [true, 'Show the titles on the console as they are grabbed', true]),
+        OptString.new('EMBY_SERVER', [true, 'Emby Web UI IP to use (e.g. 10.10.10.18))', '']),
+        OptInt.new('EMBY_PORT', [true, 'Web UI port for Emby Server (e.g. 8096)', '8096']),
+        OptString.new('PORTS', [true, 'Ports to scan (e.g. 22-25,80,110-900)', '80,8080,8081,8888'])
+      ]
```

Comment:
Need to add in a `TARGETURI`.  While very unlikely to have one, better to add it now.
https://github.com/rapid7/metasploit-framework/blob/master/modules/exploits/example_webapp.rb#L78